### PR TITLE
Update index.yml to reflect machine learning library in homepage card

### PR DESCRIPTION
**Pull request by KittyYeungQ**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1f20a948c6b26d15dc2c49b8a39d2c3f1f4ff9fd/articles/index.yml) | 87 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6f3769b-2f8c-46c1-a3aa-945d45596694_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/index.yml [hunk 1]
```diff
@@ -99,7 +99,10 @@ landingContent:
         links:
           - text: Standard libraries
             url: libraries/standard/index.md
-          - text: Numerics library
-            url: libraries/numerics/index.md
           - text: Chemistry library
             url: libraries/chemistry/index.md
+          - text: Machine learning library
```
Comment (geduardo) [inline articles/index.yml]:
```suggestion
          - text: Quantum machine learning library
```
For consistency with the TOC

Comment (KittyYeungQ) [reply articles/index.yml]: To be consistent with how the TOC and the cards are different. The shorter version as is more concise and may be better?

image

Comment (geduardo) [reply articles/index.yml]: Ok, you're right! Sorry

**Pull request merged**